File: packages/insomnia-httpsnippet/test/fixtures/output/python/python3/headers.py
import http.client

conn = http.client.HTTPConnection("mockbin.com")

headers = {
    'accept': "application/json",
    'x-foo': "Bar"
    }

conn.request("GET", "/har", headers=headers)

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))

File: packages/insomnia-httpsnippet/test/fixtures/output/node/unirest/headers.js
var unirest = require("unirest");

var req = unirest("GET", "http://mockbin.com/har");

req.headers({
  "x-foo": "Bar",
  "accept": "application/json"
});


req.end(function (res) {
  if (res.error) throw new Error(res.error);

  console.log(res.body);
});

